caluma/workflow/migrations/0003_auto_20181214_1317.py

```diff
@@ -8,7 +8,7 @@
 
 class Migration(migrations.Migration):
 
-    dependencies = [("workflow", "0001_initial")]
```

Comment:
Why the migration reordering? This looks like trouble for existing databases

Reply:
Yep something went wrong when merging the migrations. Will fixed it.

Reply:
Fixed migration issue.